plugins/org.yakindu.sct.doc.user/src/user-guide/embedded_systems_integration_guide.textile

```diff
@@ -0,0 +1,281 @@
+
+h1(#embedded_systems_integration_guide). Embedded Systems Integration Guide
+
+State machine code generated by YAKINDU Statechart Tools can effectively be used on embedded systems. The integration contains different parts, but can mostly be described in a generic way, as most steps of the integration are platform independent.
+The differences of the integration mostly stem from the selected execution semantics - cycle based or event driven - as well as the used functionalities, such as in, out and timed events. Other best practice implementation patterns can be based on whether a bare metal microcontroller or an operating system is used. So, there are different factors which may have an impact on the integration:
```

Comment:
perhaps shortly describe the purpose of this chapter ...

Reply:
:+1: